# This message encodes a relative SE(3) measurement

# Used for timestamp
std_msgs/Header header

# ID of first node
uint32 idx

# absolute pose
geometry_msgs/Pose pose